.github/workflows/ci.yml

```diff
@@ -514,7 +514,7 @@ jobs:
       enterprise-license: ${{ needs.release-versions.outputs.enterprise-license }}
       shard: '[1, 2, 3, 4, 5]'
       total-shard: 5
-      mongodb-version: "['5.0']"
+      mongodb-version: "['5.0', '7.0']"
```

Comment:
?

Reply:
since I removed the `no watcher` that uses `7.0` I moved the value to here